packages/caraml-auth-google/.gitignore

```diff
@@ -0,0 +1,5 @@
+build/
```

Comment:
Can these be moved to a root level `.gitignore`, replacing `caraml_auth_google.egg-info.egg-info/` with `*.egg-info` ?

Reply:
Hmm oh yes good point, will do that.